### PR TITLE
Improve Vietnamese translation

### PR DESCRIPTION
**Pull request by nit27**
I've improved some translations for practical context, also corrected typos.

### CODE REVIEW

**Comment by nit27** [comment]
@vynguyen94, what do you think? 🤓

File: locales/vi-VN/translation.json [hunk 1]
```diff
@@ -224,17 +224,17 @@
   "Close": "Đóng",
   "Tab navigation": "Tab điều hướng",
   "Make this your default address": "Gán thành địa chỉ ví mặc định",
-  "into": "thêm vào",
-  "out from": "ra khỏi",
+  "into": "into",
+  "out from": "out from",
```
[the remark this answers is not in the document]
Comment (nit27) [reply locales/vi-VN/translation.json:228]: `out from: ra khỏi từ` ?

[the remark this answers is not in the document]
Comment (vynguyen94) [reply locales/vi-VN/translation.json:228]: yea, this quite ok.

**Comment by vynguyen94** [comment]
All good. 
Have a good day! :)

**Comment by nop33** [comment]
Merging, since it has been reviewed and approved. Thank you @vynguyen94 and @nit27 !

**Pull request merged**